File: src/org/ggp/base/util/gdl/model/SentenceModelUtils.java
package org.ggp.base.util.gdl.model;

import java.util.Set;

import org.ggp.base.util.gdl.grammar.GdlSentence;


class SentenceModelUtils(object):
    def static bool inSentenceFormGroup(GdlSentence sentence,
            Set<SentenceForm> forms):
        for(SentenceForm form : forms)
            if(form.matches(sentence))
                return true;
        return false;

File: ggpy/cruft/autocode/SentenceModelUtils.py
#!/usr/bin/env python
""" generated source for module SentenceModelUtils """
# package: org.ggp.base.util.gdl.model
import java.util.Set

import org.ggp.base.util.gdl.grammar.GdlSentence

class SentenceModelUtils(object):
    """ generated source for class SentenceModelUtils """
    @classmethod
    def inSentenceFormGroup(cls, sentence, forms):
        """ generated source for method inSentenceFormGroup """
        for form in forms:
            if form.matches(sentence):
                return True
        return False

